inc/html.class.php

```diff
@@ -5102,6 +5102,9 @@ static function fileForRichText($options=array()) {
     *    - pasteZone           string   DOM ID of the paste zone
     *    - dropZone            string   DOM ID of the drop zone
     *    - rand                string   already computed rand value
+    *    - display             boolean  display or return the generated html
+    *
+    * @return mixed          the html if display paremeter is false or true
```

Comment:
There is a typo here, and this is unclear to me. Maybe replace mixed with "string|void"